firstvoices/firstvoices/settings.py

```diff
@@ -87,7 +87,7 @@
         # the first 2 are for admin app compatibility
         "rest_framework.authentication.SessionAuthentication",
         "rest_framework.authentication.BasicAuthentication",
-        "backend.jwt_auth.UserAuthentication",
+        # "backend.jwt_auth.UserAuthentication",
```

Comment:
Should we uncomment this ? or let it be commented till we get cognito login working ?